test/firebase_io_test.dart

```diff
@@ -30,6 +30,24 @@ void main() {
       rootValue = await fbClient.get(baseUri);
 
       expect(rootValue, putContent);
+
+
+      // TODO: Consider improving how we construct this new path.
+      var postUri = Uri
+          .parse('${baseUri.toString().split('.json').first}/interests.json');
+
+      var postContent = [
+        {'name': 'Encryption', 'love-level': 10},
+        {'name': 'Computer Science', 'love-level': 7}
+      ];
+
+      postContent.forEach((interest) async {
```

Comment:
This type of `.forEach` call won't work, since you can't await each call.

Use

```dart
for (var interest in postContent) {
  rootValue = await fbClient.post(pustUri, interest);
  // ..
)
```